libraries/AC_InputManager/AC_InputManager_Heli.cpp

```diff
@@ -119,4 +122,34 @@ float AC_InputManager_Heli::get_pilot_desired_collective(int16_t control_in)
     return collective_out;
 }
 
+// parameter_check - check if input manager specific parameters are sensible
+bool AC_InputManager_Heli::parameter_check(char* fail_msg, uint8_t fail_msg_len) const
+{
+    // returns false if STAB_COL_1 is outside of range
+    if ((_heli_stab_col_min < 0) || (_heli_stab_col_min > 100)){
+        hal.util->snprintf(fail_msg, fail_msg_len, "IM_STAB_PCT_COL_1 out of range");
```

Comment:
In no way, shape or form critical, but we could save some flash here with
```suggestion
        hal.util->snprintf(fail_msg, fail_msg_len, "%s out of range", "IM_STAB_PCT_COL_1");
```